tools/rosbash/CMakeLists.txt

```diff
@@ -8,6 +8,9 @@ install(FILES rosbash rosfish rostcsh roszsh
 if (WIN32)
   install(PROGRAMS scripts/rosrun.bat
     DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION})
+
+  install(PROGRAMS  scripts/findpath.py scripts/roscd.bat scripts/rosls.bat
```

Comment:
The filename `findpath` sounds very generic. How about renaming it to `rosfindpath`?

Nitpick: please remove one of the two spaces after `PROGRAMS`.

Reply:
thanks for the name suggestion, updated to `rosfindpath`, removed extra space